presto-native-execution/presto_cpp/main/LinuxMemoryChecker.cpp

```diff
@@ -143,10 +215,15 @@ class LinuxMemoryChecker : public PeriodicMemoryChecker {
   const boost::regex kInactiveAnonRegex{R"!(inactive_anon\s*(\d+)\s*)!"};
   const boost::regex kActiveAnonRegex{R"!(active_anon\s*(\d+)\s*)!"};
   const boost::regex kMemAvailableRegex{R"!(MemAvailable:\s*(\d+)\s*kB)!"};
-  const boost::regex kMemTotalRegex{R"!(MemTotal:\s*(\d+)\s*kB)!"};
+  const boost::regex kMemTotalRegex{R"!(MemTotal:\s*(\d+)\s+kB)!"};
   const char* kCgroupV1Path = "/sys/fs/cgroup/memory/memory.stat";
   const char* kCgroupV2Path = "/sys/fs/cgroup/memory.stat";
+  const char* kCgroupV1MaxMemFilePath =
+      "/sys/fs/cgroup/memory/memory.limit_in_bytes";
+  const char* kCgroupV2MaxMemFilePath = "/sys/fs/cgroup/memory.max";
   std::string statFile_;
+  std::string memInfoFile_ = "/proc/meminfo";
```

Comment:
Nit: `const std::string kMemInfoFile_`

Reply:
This has to stay as a non-const variable since I need to change its path to point to the meminfo test file when I am running the LinuxMemoryCheckerTests.